packages/cms-cli/commands/functions/build.js

```diff
@@ -0,0 +1,73 @@
+const ora = require('ora');
+const {
+  addPortalOptions,
+  addConfigOptions,
+  setLogLevel,
+  getPortalId,
+  addUseEnvironmentOptions,
+} = require('../../lib/commonOpts');
+const { trackCommandUsage } = require('../../lib/usageTracking');
+const { logDebugInfo } = require('../../lib/debugInfo');
+const {
+  loadConfig,
+  validateConfig,
+  checkAndWarnGitInclusion,
+} = require('@hubspot/cms-lib');
+const { logger } = require('@hubspot/cms-lib/logger');
+const { buildPackage } = require('@hubspot/cms-lib/api/functions');
+const { validatePortal } = require('../../lib/validation');
+
+const makeSpinner = (functionPath, portalIdentifier) => {
+  return ora(
+    `Building new bundle for '${functionPath}' on portal '${portalIdentifier}'.\n`
+  );
+};
+
+const loadAndValidateOptions = async options => {
+  setLogLevel(options);
+  logDebugInfo(options);
+  const { config: configPath } = options;
+  loadConfig(configPath, options);
+  checkAndWarnGitInclusion();
+
+  if (!(validateConfig() && (await validatePortal(options)))) {
+    process.exit(1);
+  }
+};
+
+exports.command = 'build <path>';
+exports.describe =
```

Comment:
I wonder if this needs more/different description.  I don't know how to succinctly say it for the CLI, but what I am wondering is, is someone going to understand what is happening here vs actually deploying the function?  Will they know this is a required step?  Once this action happens, the user doesn't have any insight into what just happened, what it really means, or why it is required.  

This may be a larger discussion, it just seems a little odd to me.  Not saying we should do this now, but maybe it would be helpful to add capabilities to know the status of the current deployable unit.

Like be able to see what version or identifier of the bundle is ready to be deployed.  Maybe also allow them to download a deployable bundle to make rollbacks/backups easy and smooth

Reply:
I totally agree, and there will likely be more steps in the future to complete the deploy process(which we can discuss streamlining). This is only part of an alpha release so we can get more feedback on how the preferred solution should function.